app/services/data_migrations/delete_uuidless_courses.rb

```diff
@@ -0,0 +1,17 @@
+module DataMigrations
+  class DeleteUuidlessCourses
+    TIMESTAMP = 20210601155313
+    MANUAL_RUN = false
```

Comment:
since we're raising an error, I think this migration might need to be manual to avoid breaking the deploy

Reply:
Maybe best to run manually in order to ensure that it has run, however, It shouldn't break the deploy as the rake task will handle any exceptions

https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/lib/tasks/data.rake#L23

Reply:
ah that's great! but yea preferably manually just to help with visibility